DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/GroupCoursesController.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates
+{
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ServiceFilter;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupCourses;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+
+    [FeatureGate(FeatureFlags.RefactoredTrackingSystem)]
+    [Authorize(Policy = CustomPolicies.UserCentreAdmin)]
+    [SetDlsSubApplication(nameof(DlsSubApplication.TrackingSystem))]
+    [SetSelectedTab(nameof(NavMenuTab.Delegates))]
+    [Route("TrackingSystem/Delegates/Groups/{groupId:int}/Courses")]
+    public class GroupCoursesController : Controller
+    {
+        private readonly ICourseService courseService;
+        private readonly IGroupsService groupsService;
+        private readonly IUserService userService;
+
+        public GroupCoursesController(
+            IUserService userService,
+            ICourseService courseService,
+            IGroupsService groupsService
+        )
+        {
+            this.userService = userService;
+            this.courseService = courseService;
+            this.groupsService = groupsService;
+        }
+
+        [Route("{page:int=1}")]
+        public IActionResult Index(int groupId, int page = 1)
+        {
+            var centreId = User.GetCentreId();
+            var groupName = groupsService.GetGroupName(groupId, centreId);
+
+            if (groupName == null)

```

Comment:
If decorating the controller with `VerifyAdminUserCanAccessGroup` it should be same to drop this.